src/tabs/AOCTab/days/day3/Day3Solution.js

```diff
@@ -1,14 +1,136 @@
 const Day3Solution = (props) => {
+    // Parses file, separating string into individual values
   const input = props.input
-  console.log("Day 3 input: " + input); 
+  var binary_input = input.split("\r\n");
+
+  // Lengths of binary digits and array
+  const width = binary_input[0].length; 
+  const height = binary_input.length;
+
+  let binary_input_gamma = "";
+  let binary_input_epsilon = "";
+
+  for (let c = 0; c < width; c++) {
+    /**
+     * Find most common bit in gamma rate
+     * Access each individual value in list
+     */
+    let count_0 = 0;
+    let count_1 = 0;
+    /**
+     * Calculates the number of times first bit of each number is present 
+     */ 
+    for (let r = 0; r < height; r++) {
+      parseInt(binary_input[r][c]) === 0 ? count_0 += 1 : count_1 += 1;

```

Comment:
`parseInt` is always supposed to take in a second argument for what base the string is in! so i know that it doesn't technically matter in this case, but if you had ESLint configured it would complain

```suggestion
      parseInt(binary_input[r][c], 2) === 0 ? count_0 += 1 : count_1 += 1;

```